edge_api/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+server.port=8050
+
+instrumentservice.baseurl= ${INSTRUMENT_SERVICE_BASEURL:localhost:8051}
```

Comment:
Ik denk dat de ports niet gaan werken op okteto, dus hier zou je best ook een env variabele toevoegen. Dit moeten we misschien eerst even in een lokale cluster testen.